packages/cli/src/utils/config.ts

```diff
@@ -28,11 +17,18 @@ export function configPath(configDir: string) {
 
 export function readConfig(configDir: string): CeloConfig {
   if (fs.pathExistsSync(configPath(configDir))) {
-    const existingConfig = fs.readJSONSync(configPath(configDir))
+    const { gasCurrency, existingConfig } = fs.readJSONSync(configPath(configDir))
     const combinedConfig = { ...defaultConfig, ...existingConfig }
     if (combinedConfig.hasOwnProperty('nodeUrl')) {
       combinedConfig.node = combinedConfig.nodeUrl
     }
+
+    // NOTE: make sure we don't confuse the user by printing a gasCurrency
+    // that's not being used
+    if (gasCurrency) {
+      writeConfig(configDir, combinedConfig)
```

Comment:
can we solve the issue of not printing without writing to the config during the read op? 

I think writing while reading can lead to weird bugs generally. 

I propose instead of sanitize out the gasCurrency from the returned config

Reply:
That's what I did initially but then if ppl open the file manually (for whatever reason) and still see a feecurrency in there, may be confusing

Reply:
i feel like people arent gonna be doing that

Reply:
fair enough.